Classes/Emogrifier.php

```diff
@@ -530,6 +530,13 @@ private function fillStyleAttributesWithMergedStyles()
     private function generateStyleStringFromDeclarationsArrays(array $oldStyles, array $newStyles)
     {
         $combinedStyles = array_merge($oldStyles, $newStyles);
+
+        foreach ($oldStyles as $attribute => $expression) {
```

Comment:
For better readability, please let's move this to a well-named private method. I propose the name "applyImportantStyles".